src/components/ui-library/Slider.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid } from '@material-ui/core';
+import { withStyles } from '@material-ui/styles';
+import { spacing, Dot } from '../styledComponents';
+
+const styles = {
+  controllers: { marginTop: spacing(5) },
+};
+
+const stringify = (element) => {
+  let res = '';
+  for (let index = 0; index < element.length; index += 1) res += element[index].key;
+  return res;
```

Comment:
There's nothing wrong with using a `for` loop like this, but I'll generally use [Array.reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) for this sort of thing.

Reply:
Got it 🙃